Formula/mutt.rb

```diff
@@ -47,18 +46,19 @@ def install
     user_admin = Etc.getgrnam("admin").mem.include?(ENV["USER"])
 
     args = %W[
+      --disable-debug
```

Comment:
No, please don't do this.  It's common to need to run `mutt` in debug mode to troubleshoot connection problems (pop/imap/smtp), particularly when communicating with upstream about apparent problems.  If you're going to remove the option, then just pass `--enable-debug`.

Reply:
Is there a downside to having `--enable-debug` enabled by default?

Reply:
None that I have ever found in 13 years of mutting.